python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -621,7 +627,8 @@ cdef class ParquetWriter:
                   int max_page_size_bytes=524288,
                   int max_page_size_rows=20000,
                   int max_dictionary_size=1048576,
-                  bool use_dictionary=True):
+                  bool use_dictionary=True,
+                  bool store_schema=True):
```

Comment:
Should we change the internal arg names `write_arrow_schema` to `store_schema` (used in pyarrow) or let it be?